ros_buildfarm/status_page.py

```diff
@@ -805,35 +792,21 @@ def _get_blocked_releases_info(config_url, rosdistro_name, repo_names=None):
                 unreleased_pkgs = unreleased_pkgs.difference(packages)
 
                 # Get maintainer info and repo of unreleased packages
-                maintainers = {}
+                maintainers = defaultdict(dict)
                 repos_blocked_by = set()
                 for pkg_name in unreleased_pkgs:
                     unreleased_repo_name = \
                         prev_distro_file.release_packages[pkg_name].repository_name
                     repos_blocked_by.add(unreleased_repo_name)
-                    pkg_xml = prev_distribution.get_release_package_xml(pkg_name)
-                    if pkg_xml is not None:
-                        try:
-                            pkg = parse_package_string(pkg_xml)
-                        except InvalidPackage:
-                            pass
-                        else:
-                            pkg_maintainers = {m.name: m.email for m in pkg.maintainers}
-                            if unreleased_repo_name not in maintainers:
-                                maintainers[unreleased_repo_name] = {}
-                            maintainers[unreleased_repo_name].update(pkg_maintainers)
+                    new_maintainers = dict(_maintainers(prev_distribution, pkg_name))
+                    maintainers[unreleased_repo_name].update(new_maintainers)
```

Comment:
Maybe eliminate the single use var `new_maintainers`?

Reply:
Removed single use variable and squashed into first commit fc9041c .